XS.xml

```diff
@@ -0,0 +1,25 @@
+<crossSection base="=== test ===">
```

Comment:
This file should be removed... Is not needed since you are going to add a real test